docs/checkout.md

```diff
@@ -0,0 +1,52 @@
+# Checkout
+
+## Google Documentation
+
+https://developers.google.com/tag-manager/enhanced-ecommerce#checkout
+
+## Introduction
+
+The checkout steps are registered from the backend code. Based on the controller / method a certain step will be triggered and sent to GTM.
+
+The checkout options are tracked through JS. The payment & shipping forms are being listened for submits, and at that moment the
+selected option will be sent to GTM. Keep this in mind when adjusting the checkout forms.
+
+## Configuration in GTM
+
```

Comment:
I suggest adding some sort or remark and/or saparation here, to clarify that:

1. this is separate than the potentially existing `pageview` tag & trigger.
2. these are 2 sets of a tag and a trigger each

Reply:
Made some improvements. WDYT now?